### ISSUE TITLE
Macports AMUSE package

### ISSUE THREAD

**Issue by rieder** [amusecode/amuse#526]
**Is your feature request related to a problem? Please describe.**
Macports is our suggested way of installing AMUSE prerequisites on macOS. Offering AMUSE as Macports packages would therefore allow us to install AMUSE in one go with the prerequisites.

**Describe alternatives you've considered**
Just offering pip packages works, but if we install Amuse as a Macports package the prerequisites will automatically be managed, which will also allow binary packages.

**Additional context**
Analogue to #294 and #525

**Comment by rieder** [comment]
In Macports, there is the "variants" option that we can use to specify the MPI flavour to use.
E.g. "port install amuse-framework +openmpi" or "port install amuse-framework +mpich", where either of these would be the default. Other AMUSE packages would have to use this same variant.

**Comment by rieder** [comment]
https://guide.macports.org/#development

**Comment by rieder** [comment]
This seems like it's very doable so I'll try to make a package for amuse-framework soon.

**Comment by ipelupessy** [comment]
I think @merijn had a homebrew package...

**Comment by merijn** [comment]
Yeah, I made a homebrew package which is here: https://github.com/nlesc/homebrew-nlesc

It's, uh, beta quality ;)

**Comment by rieder** [comment]
Let's keep Homebrew in a separate issue though (just opened #527), it's quite different from macports.

**Comment by rieder** [comment]
Here's a first version of a Portfile for amuse-framework. It seems to work for me but is probably still lacking in many ways (for instance, many versions are hard-coded).
https://gist.github.com/rieder/b065f4ec562f0e93a3891c5db8cd9c1f
